components/create_post/create_post.jsx

```diff
@@ -369,14 +370,17 @@ export default class CreatePost extends React.Component {
         });
     }
 
-    sendReaction(isReaction) {
+    sendReaction = async (isReaction) => {
         const channelId = this.props.currentChannel.id;
         const action = isReaction[1];
         const emojiName = isReaction[2];
         const postId = this.props.recentPostIdInChannel;
 
         if (postId && action === '+') {
-            this.props.actions.addReaction(postId, emojiName);
+            const {data} = await this.props.actions.addReaction(postId, emojiName);
+            if (data) {
```

Comment:
Do we need to wait for the addReaction to be successful? We've already checked that the emoji exists, so I don't think we need to wait for that as well. Plus, I'm not sure if marking this whole function async is going to have any other side effects.

Reply:
0/5 with my implementation. I just initially thought that emoji/reaction should only be added to recent emojis if reaction is added successfully.